lib/rules/web/admin_console/4.2/navigator_menu.xyaml

```diff
@@ -0,0 +1,12 @@
+expand_monitoring:
```

Comment:
How about adding a common rule with name `expand_primary_menu` then it could be reused?